msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -443,16 +443,31 @@ For more information on cards and cards in bots, see [cards documentation](~/tas
 
 Following are the status codes and their error code and message values:
 
-| Status code | Error code and message values | Description |
-|----------------|-----------------|-----------------|
-| 403 | **Code**: `ConversationBlockedByUser` <br/> **Message**: User blocked the conversation with the bot. | User blocked the bot in 1:1 chat or a channel through moderation settings. |
-| 403 | **Code**: `BotNotInConversationRoster` <br/> **Message**: The bot isn't part of the conversation roster. | The bot isn't part of the conversation. |
-| 403 | **Code**: `BotDisabledByAdmin` <br/> **Message**: The tenant admin disabled this bot. | Tenant blocked the bot. |
-| 401 | **Code**: `BotNotRegistered` <br/> **Message**: No registration found for this bot. | The registration for this bot wasn't found. |
-| 412 | **Code**: `PreconditionFailed` <br/> **Message**: Precondition failed, please try again. | A precondition failed on one of our dependencies due to multiple concurrent operations on the same conversation. |
-| 404 | **Code**: `ConversationNotFound` <br/> **Message**: Conversation not found. | The conversation wasn't found. |
-| 413 | **Code**: `MessageSizeTooBig` <br/> **Message**: Message size too large. | The size on the incoming request was too large. |
-| 429 | **Code**: `Throttled` <br/> **Message**: Too many requests. Also returns when to retry after. | Too many requests were sent by the bot. For more information, see [rate limit](~/bots/how-to/rate-limit.md). |
+| Status code | Error code and message values | Description | Should retry request | Developer Action |
+|----------------|-----------------|-----------------|----------------|----------------|
+| 403 | **Code**: `ConversationBlockedByUser` <br/> **Message**: User blocked the conversation with the bot. | User blocked the bot in 1:1 chat or a channel through moderation settings. | No | Remove conversation from cache. Stop attempting to post conversation until interaction with bot is explicitly initiated by a user in the conversation, indicating bot is no longer blocked. |
```

Comment:
> Remove conversation from cache. Stop attempting to post conversation until interaction with bot is explicitly initiated by a user in the conversation, indicating bot is no longer blocked. 

Since we don't have an unblock notification event, does this recommendation make the behavior unpredictable? If a user blocks the bot, then unblocks it, notifications will not resume until the user messages the bot. But the user doesn't know that this step is required. And the app can't tell the user either, because they can't message the user while the bot is blocked.

Reply:
Adrian, what do you think would be a better recommendation here?